aws/dust/README.md

```diff
@@ -0,0 +1,46 @@
+# CrashOverride Dust Extension
+
+This module can be used to query CrashOverride dust extension ARN
+to be as a lambda layer.
+
+## Example
+
+```terraform
+module "dust" {
+  source = "github.com/crashappsec/terraform-modules//aws/dust?ref=main"
```

Comment:
nit: typo

Reply:
this file is fully auto-generated from the `main.tf` below. replied there